addons/website/static/src/snippets/s_dynamic_snippet/000.scss

```diff
@@ -22,15 +22,15 @@
             @include media-breakpoint-up(lg) {
                 // Switch to a fluid layout in order to keep entries readable.
                 // Our target grid is 2columns based
-                flex: 0 1 MAX($-entry-min-width, 50%);
+                flex: 0 1 MAX(var(--DynamicEntry-min-width, #{$-entry-min-width}, 50%));
```

Comment:
According to the guidelines, the name of this variable should be something like `DynamicSnippet__entry-maxWidth`, where:

`DynamicSnippet`= JS component
`entry`= inner element identifier 
`maxWidth`= property

I'm aware that sometimes we miss to follow the convention (including me 🙋‍♂️😅).. However, as we expand our use of declarative JS (see the new widget system), we should aim to ensure a correlation with declarative CSS variables since, unlike SCSS ones, CSS variables are directly accessible from the DOM.